docs/developer-docs/cost-estimations-and-examples.md

```diff
@@ -0,0 +1,176 @@
+# Cycles cost estimations and examples
+
+
+## Overview
+
+To get a rough estimate of how much your project may cost, below are a few common project architectures and their estimated monthly cost in cycles. These estimates are broken down into the following transaction groups:
+
+
+- **Messaging**: A message transaction refers to calls that are made to a canister's methods. Messaging costs depend on the type of call being sent (query, update, inter-canister, etc), the size of the message's request and response bytes, and the total amount of messages a canister sends and receives.
+
+- **Execution**: An execution transaction refers to tasks that a canister executes and the number of instructions executed per task and per call.
+
+- **Storage**: A storage transaction refers to the amount of data the canister stores.
+
+- **Special features**: Special feature transactions include HTTPS outcalls, transactions using the [Bitcoin API](./integrations/bitcoin/bitcoin-how-it-works.md), and transactions using the [chain-key signing API](./integrations/t-ecdsa/t-ecdsa-how-it-works.md).
```

Comment:
Let's drop the "transactions" to avoid confusion with real transcations in blockchains.

Reply:
Changed "Transaction" to "Transmission"